test/mod_bosh_SUITE.erl

```diff
@@ -0,0 +1,81 @@
+-module(mod_bosh_SUITE).
+-compile([export_all]).
+-include_lib("eunit/include/eunit.hrl").
+-define(PORT, 5280).
+-define(IP, {127, 0, 0, 1}).
+-define(CUSTOM_HEADERS,
+        [{<<"strict-transport-security">>, <<"max-age=31536000; includeSubDomains">>},
+         {<<"access-control-allow-origin">>, <<"*">>}]).
+-define(BOSH_BIND_PATH, "/http-bind").
+-define(HANDSHAKE_TIMEOUT, 3000).
+-define(eq(E, I), ?assertEqual(E, I)).
+-define(FAST_PING_RATE, 500).
```

Comment:
Some of these defines can be removed. I suspect it's a direct copy & paste from websockets suite? :)

Reply:
It is indeed